ts/features/fims/screens/FimsWebviewScreen.tsx

```diff
@@ -68,7 +70,24 @@ const FimsWebviewScreen = () => {
       );
       return;
     }
-    const url = new URLParse(maybeParams.value.url as string, true);
+
+    const maybeParsedUrl = maybeFimsDomain.chain(domain => {
+      const parsed = new URLParse(domain as string, true);
+      return parsed.protocol ? some(parsed) : none;
```

Comment:
Even with this check, putting a wrong protocol like `hhp://127.0.0.1` will generate a native error. Do you think it's worth adding a check against the protocol? If it's a case that won't really happen feel free to mark this comment as resolved! 🙏

Reply:
This should not happen since it is under our control, but a check makes it more safe!